dead.py

```diff
@@ -17,21 +17,26 @@
 from identify.identify import tags_from_path
 
 UsageMap = DefaultDict[str, Set[str]]
+DefineSet = Set[Tuple[str, str, int]]
+DisableSet = Set[Tuple[str, int]]
```

Comment:
there's no point in making a type alias if it's only used once

Reply:
fixed